flask/cli.py

```diff
@@ -725,7 +725,9 @@ def _validate_key(ctx, param, value):
     return value
 
 
-@click.command('run', short_help='Runs a development server.')
+@click.command('run',
+               short_help='Runs a development server (flask run --help for '
```

Comment:
It doesn't make sense to add this here but not to any other commands. We're following the general pattern that `--help` is commonly understood to exist for command line tools, and the usage message also calls this out on bad args.